charts/trino/templates/_helpers.tpl

```diff
@@ -120,3 +120,19 @@ Code is inspired from bitnami/common
   {{- end -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Create the name of the file auth secret to use
+*/}}
+{{- define "trino.fileAuthSecretName" -}}
+{{- if and .Values.auth .Values.auth.passwordAuthSecret }}
+{{- .Values.auth.passwordAuthSecret | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}-file-authentication
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}-file-authentication
```

Comment:
If we append an extra 20 characters, shouldn't we use `trunc 43`?

Reply:
There were a number of places where the string is truncated to 63 but then is appended to. I've amended all of them in the new commit, ensuring that truncation is always done _after_ the string is built.
Hopefully this will prevent the issue reoccurring if there are any future renames.